src/psyclone/tests/psyir/symbols/symboltable_test.py

```diff
@@ -136,6 +137,79 @@ def test_add():
             "'var1'.") in str(error.value)
 
 
+def test_imported_symbols():
+    ''' Test the imported_symbols method. '''
+    sym_table = SymbolTable()
+    my_mod = ContainerSymbol("my_mod")
+    sym_table.add(my_mod)
+    assert sym_table.imported_symbols(my_mod) == []
+    var1 = DataSymbol("var1", DataType.REAL, interface=LocalInterface())
+    sym_table.add(var1)
+    assert sym_table.imported_symbols(my_mod) == []
+    var2 = DataSymbol("var2", DataType.INTEGER,
+                      interface=GlobalInterface(my_mod))
+    assert sym_table.imported_symbols(my_mod) == []
+    sym_table.add(var2)
+    assert sym_table.imported_symbols(my_mod) == [var2]
+    var3 = DataSymbol("var3", DataType.INTEGER,
+                      interface=GlobalInterface(my_mod))
+    sym_table.add(var3)
+    imported_symbols = sym_table.imported_symbols(my_mod)
+    assert var3 in imported_symbols
+    assert var2 in imported_symbols
+    # Passing something that is not a ContainerSymbol is an error
+    with pytest.raises(TypeError) as err:
+        sym_table.imported_symbols(var2)
+    assert "expects a ContainerSymbol but got an object of type" in \
+        str(err.value)
+
+
+def test_remove():
+    '''Test that the remove method removes symbols from the symbol
+    table, but raises appropiate errors when provided with wrong
```

Comment:
Update text to say that it raises and exception if symbols exist in the container. Also point out this is just testing container symbols at the moment.

Reply:
Good point. Done.

Reply:
There is also a small typo "appropiate"

Reply:
Done.